include/lemlib/devices/encoder/optical.hpp

```diff
@@ -21,7 +21,7 @@ class OpticalEncoder : public Encoder {
          *
          * @return float angle rotated by the optical encoder, in radians
```

Comment:
```suggestion
         * @return Angle angle rotated by the optical encoder
```
Update doxygen comment